honeybee/hbfensurface.py

```diff
@@ -48,14 +48,12 @@ class HBFenSurface(HBAnalysisSurface):
         hbsrf.write_rad_file(r"c:/ladybug/triangle.rad", include_materials=True)
     """
 
-    def __init__(self, name, sorted_points=None, is_name_set_by_user=False,
-                 rad_properties=None, ep_properties=None, states=None):
+    def __init__(self, name, sorted_points=[], is_name_set_by_user=False,
```

Comment:
Using mutable default value as an argument is not good practice and is against PEP8 if I'm not mistaken. 
Best practice is using None as we do. Here is why: 

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Of course you're are right, I had just done a global search and replace and seemed to go overboard here.